### DISCUSSION TITLE
Maximum Filter Size

### DISCUSSION THREAD

**Discussion by Aurifex01** [keras-team/autokeras#1356] [General]
Hey,
is it possible to set a maximum filter size?
Everytime autokeras tries huge filter sizes of 256 or 512 i am runnning out of memory (VRAM).

**Comment by haifeng-jin** [comment]
I assume you mean the number of filters instead of the filter size. The filter size is no larger than (7, 7).

The RAM not likely to be an issue in our next release. We will reduce the batch_size automatically when running out of memory.
Moreover, you can set the [max_model_size](https://github.com/keras-team/keras-tuner/blob/master/kerastuner/engine/tuner.py#L40) to control the model size.
